src/nina_vesseltun_ex/ex04/myrand.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Nina Mariann Vesseltun'
+__email__ = 'nive@nmbu.no'
+
+
+from random import randint
+
+
+class ListRand:
+    def __init__(self, liste):
+        self.liste = liste
+        self.current_idx = -1
+
+    def rand(self):
+        if self.current_idx == len(self.liste) - 1:
+            raise RuntimeError
+        else:
+            self.current_idx += 1
+            return self.liste[self.current_idx]
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+
+    def rand(self):
+        a = 16807
+        m = 2 ** 31 - 1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime `rand()` is called